src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -262,7 +264,9 @@
 
       <AspNetCore31RuntimePackRids Include="@(AspNetCore30RuntimePackRids)" />
       <AspNetCore50RuntimePackRids Include="@(AspNetCore31RuntimePackRids);linux-musl-arm;win-arm64" />
-      <AspNetCoreRuntimePackRids Include="@(AspNetCore50RuntimePackRids);osx-arm64;linux-s390x" />
+      <AspNetCoreRuntimePackRids Include="@(AspNetCore50RuntimePackRids);osx-arm64" />
```

Comment:
I think we can delete this line now?

Reply:
> I haven't deleted it as this variable is used in KnownFrameworkReference for net8.0

In that case, perhaps this should mean `AspNetCore80RuntimePackRids`? And inherit from `70`?

Reply:
@omajid Added AspNetCore80RuntimePackRids inherited from 70, please check